data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -513,6 +561,15 @@ def _workforce_factor(self) -> bool:
         # Where the high school degree achievement rates for adults 25 years and older is less than 95%
         # (necessary to screen out university block groups)
 
+        # Workforce criteria for states fields.
+        workforce_eligibility_columns = [
+            field_names.UNEMPLOYMENT_LOW_HS_EDUCATION_FIELD,
+            field_names.POVERTY_LOW_HS_EDUCATION_FIELD,
+            field_names.LINGUISTIC_ISOLATION_LOW_HS_EDUCATION_FIELD,
+            field_names.MEDIAN_INCOME_LOW_HS_EDUCATION_FIELD,
+            field_names.LOW_READING_LOW_HS_EDUCATION_FIELD,
```

Comment:
Added in reading, too.

Reply:
Why did we choose 3rd grade for literacy? I did some EDA last night and have some comments on this.

Reply:
It's what was easily accessible in the COI data!

Reply:
Understood - I was just curious about `Third grade math proficiency | Percentage third graders` and the relationships with high school graduation and/or bachelors degrees attainment.